wire-ios-data-model/Tests/Source/ManagedObjectContext/DatabaseMigrationTests.swift

```diff
@@ -309,7 +309,7 @@ final class DatabaseMigrationTests: DatabaseBaseTest {
 
     func testThatItPerformsMigrationFrom_Between_2_24_1_and_PreLast_ToCurrentModelVersion() {
         // NOTICE: When a new version of data model is created, please increase the last number of the array.
-        let allVersions = ["2-24-1"] + [(25...31), (39...57), (59...101)].joined().map {
+        let allVersions = ["2-24-1"] + [(25...31), (39...57), (59...102)].joined().map {
```

Comment:
just to know, why do we have the 3 ranges here after `["2-24-1"] +`?

Reply:
I don't know. I found a [PR](https://github.com/wireapp/wire-ios-data-model/pull/1032) when those 3 ranges were implemented but there is no description. Maybe @johnxnguyen remembers.

Reply:
Maybe because there are gaps between 31 and 39, 57 and 59